BossMod/Autorotation/UIRotationWindow.cs

```diff
@@ -68,9 +65,14 @@ public override void Draw()
                 {
                     ImGui.SameLine();
                     using var style = ImRaii.PushColor(ImGuiCol.Text, 0xff00ffff);
-                    UIMisc.HelpMarker(() => "You have a preset activated, which fully overrides the plan!", FontAwesomeIcon.ExclamationTriangle);
+                    UIMisc.HelpMarker(() => "You have a preset activated, which fully overrides the CD plan!", FontAwesomeIcon.ExclamationTriangle);
                 }
             }
+
+            if (ImGui.Button("Open Boss Timeline"))
```

Comment:
I think i'll just remove this from here, and move to module viewer ui. It's really rarely useful.